### DISCUSSION TITLE
23.12.06 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#83] [Daily Scrum]
.

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * PR 리뷰
* 내일은 어떤 일을 하나요?
  * 이전에 못다한 쿼리 최적화 진행

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 나에게 친구 요청 보낸 회원 조회 기능 테스트 작성
- 내일은 어떤 일을 하나요?
    - 나와 서로 친구인 회원 조회 기능 테스트 작성

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 하나요?
  - 음악 저장 기능 수정
  - 플레이리스트에 음악 저장 기능 테스트 작성 중
- 내일은 어떤 일을 하나요? 
  - 테스트 케이스 정리
  - 플레이리스트 음악 저장 기능 완료
